indico/modules/events/contributions/forms.py

```diff
@@ -187,6 +188,9 @@ class ContributionTypeForm(IndicoForm):
     """Form to create or edit a ContributionType"""
 
     name = StringField(_("Name"), [DataRequired()])
+    is_private = BooleanField(_("Private contribution type"), widget=SwitchWidget(), default=False,
```

Comment:
Maybe "Private" or "Internal" is a better title? "Private contribution type" sounds a lot like something that'll be wrapped on two lines..